controllers/etcd_controller_test.go

```diff
@@ -72,8 +72,8 @@ var (
 	clientPort              int32 = 2379
 	serverPort              int32 = 2380
 	backupPort              int32 = 8080
-	imageEtcd                     = "eu.gcr.io/gardener-project/gardener/etcd:v3.4.13"
-	imageBR                       = "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.11.1"
+	imageEtcd                     = "eu.gcr.io/gardener-project/gardener/etcd:v3.4.13-bootstrap"
+	imageBR                       = "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.12.0-dev-2cb339762d03d5cc9c78bda855fdf48f309de9c3"
```

Comment:
```suggestion
	imageBR                       = "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.12.0"
```

Reply:
I can't make this change yet since `etcdbrctl:v0.12.0` has not yet been released, so the tests would fail. Have used the latest etcdbr version from master at the moment